src/compressed_tensors/compressors/quantized_compressors/pack_quantized.py

```diff
@@ -220,30 +220,34 @@ def pack_to_int32(
     if num_bits < 1:
         raise ValueError(f"num_bits must be at least 1, got {num_bits}")
 
-    # convert to unsigned for packing
+    # Convert to unsigned range for packing, matching quantization offset
     offset = 1 << (num_bits - 1)
     value = (value + offset).to(torch.uint8)
-    value = value.cpu().numpy().astype(np.uint32)
```

Comment:
Why was this line removed?

Reply:
> I'm not confident that we can just skip the logic. I still have to generate the right shape for the weight for the loading to work and seems like numpy can't convert meta device type.

Right, so numpy and cpu-specific logic should be converted to support torch meta tensors. I'm more confused why the `.astype(np.uint32)` conversion was removed, instead of being converted to be torch compatible.